packages/test-vectors/src/generators/LEIevidenceDocument.js

```diff
@@ -0,0 +1,93 @@
+const faker = require('faker');
+
+const getLEIevidenceDocument = () => {
+  const example = {
+    '@context': ['https://w3id.org/traceability/v1'],
+    type: 'LEIevidenceDocument',
+    "data": {
+      "type": faker.lorem.word(),
+      "id": faker.random.uuid(),
+      "attributes": {
+        "lei": faker.lorem.word(),
```

Comment:
is the is really a word?

Reply:
Updated, this is 20 alphanumeric characters